ot/lp/__init__.py

```diff
@@ -94,12 +94,15 @@ def emd(a, b, M, numItermax=100000, log=False):
     if len(b) == 0:
         b = np.ones((M.shape[1],), dtype=np.float64) / M.shape[1]
 
-    G, cost, u, v = emd_c(a, b, M, numItermax)
+    G, cost, u, v, resultCode = emd_c(a, b, M, numItermax)
+    resultCodeString = checkResult(resultCode)
```

Comment:
you need to choose your side. In Python variables are snake_case not CamelCase. To be consistent with recent changes in package you should write:

result_code_string = check_result(result_code)

I know there is the C++ underneath that use CamelCase but in the binding you should adopt the conventions of the rest of the package.

my 2c

Reply:
Right, thank you I didn't notice. I also replaced numItermax with num_iter_max and applied the same changes to the cython wrapper.